paseos/actors/base_actor.py

```diff
@@ -33,37 +34,38 @@ class BaseActor(ABC):
     _central_body = None
 
     # Communication links dictionary
-    _communication_links = DotMap(_dynamic=False)
+    _communication_devices = DotMap(_dynamic=False)
 
-    def __init__(
-        self, name: str, position, velocity, epoch: pk.epoch, central_body: pk.planet
-    ) -> None:
+    def __init__(self, name: str, position, epoch: pk.epoch) -> None:
         """Constructor for a base actor
 
         Args:
             name (str): Name of this actor
             position (list of floats): [x,y,z]
-            velocity (list of floats): [vx,vy,vz]
             epoch (pykep.epoch): Epoch at this pos / velocity
-            central_body (pk.planet): pykep central body
         """
         logger.trace("Instantiating Actor.")
-        BaseActor._check_init_value_sensibility(position, velocity)
         super().__init__()
         self.name = name
         self._local_time = epoch
-        self._central_body = central_body
-        self._orbital_parameters = pk.planet.keplerian(
-            epoch,
-            position,
-            velocity,
-            central_body.mu_self,
-            1.0,
-            1.0,
-            1.0,
-            name,
-        )
-        self._communication_links = DotMap(_dynamic=False)
+
+        self._communication_devices = DotMap(_dynamic=False)
+
+    def get_local_time(self) -> pk.epoch:
```

Comment:
Is it not more natural to use getters? See this conversation in Stackoverflow: https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters#:~:text=%2B500-,What%27s%20the%20pythonic%20way%20to%20use%20getters%20and%20setters%3F,-The%20%22Pythonic%22%20way

Reply:
changed everywhere :) forgot about this feature, thanks for pointing out!